packages/react-magma-dom/src/components/AppBar/AppBar.stories.tsx

```diff
@@ -73,3 +76,32 @@ Tabs.args = {
   ...Default.args,
   children: null,
 };
+
+export const WithSkipLink = () => {
+  return (
+    <div style={{ height: '200px', overflow: 'auto', position: 'relative' }}>
+      <AppBar isInverse position={AppBarPosition.sticky}>
+        AppBar content
+      </AppBar>
+      <SkipLink to="#last-one">Skip to the end!</SkipLink>
```

Comment:
Looking at this made me realize our SkipLink component is weird. It requires children (but only because it extends Hyperlink) and it shouldn't. The text that shows up on the button is what's passed to the `buttonText` prop and not what's passed to the children
![image](https://github.com/user-attachments/assets/cb2c7ac9-9342-4449-83ac-c5c069b1fe8f)

Reply:
Now that it doesn't require children, we can update this example to be self closing.